python/rmm/tests/test_rmm.py

```diff
@@ -216,6 +216,26 @@ def test_rmm_device_buffer_copy_from_host(hb):
     np.testing.assert_equal(expected, result)
 
 
+def test_rmm_device_buffer_copy_from_device():
+    cuda_ary = rmm.DeviceBuffer.to_device(b"abc")
+    db = rmm.DeviceBuffer.to_device(np.zeros(10, dtype="u1"))
+    db.copy_from_device(cuda_ary)
+
+    expected = np.array([97, 98, 99, 0, 0, 0, 0, 0, 0, 0], dtype="u1")
+    result = db.copy_to_host()
+
+    np.testing.assert_equal(expected, result)
+
+    cuda_ary = rmm.to_device(np.array([97, 98, 99], dtype="u1"))
```

Comment:
Should we use `pytest.parametrize` to combine these two?

Reply:
Marking resolved as this is already being discussed in [this thread]( https://github.com/rapidsai/rmm/pull/275#discussion_r377986498 ).